sources/Inputs/include/Behaviours.h

```diff
@@ -178,9 +178,10 @@ struct StaticVarCompensator {
    * @param USetPointMax the high voltage set point of the SVarC
    * @param b0 the initial susceptance value  of the SVarC
    * @param slope the slope (kV/MVar) of the SVarC
+   * @param useDefaultModel true if default model is going to be used for the SVarC
```

Comment:
From DFL point of view, it is weird to have the information "use default model" connected to the action "do nothing", even if is relevant from dynawo point of view. Replace default model by something like "exportSvarcModel"

Reply:
What about `defineSvarcModel` ?

Reply:
OK

Reply:
Done. I will use the same semantics for the new flag that I'm adding for the generators. I plan to include it in this  PR.